imports/ui/Form.jsx

```diff
@@ -169,10 +180,10 @@ export default class Form extends Component {
       // if (!error && result === 1) {
       if (error || result === 1) {
         // Success, so reset form
-        ReactDOM.findDOMNode(this.refs.ratingSlider).value = 3;
-        ReactDOM.findDOMNode(this.refs.diffSlider).value = 3;
-        ReactDOM.findDOMNode(this.refs.workloadSlider).value = 3;
-        ReactDOM.findDOMNode(this.refs.profSelect).value = "none";
+        this.ratingSlider.current.value = 3;
+        this.diffSlider.current.value = 3;
+        this.workloadSlider.current.value = 3;
+        this.profSelect.current.value = "none";
```

Comment:
This is a good first step, but I still believe these types of imperative DOM manipulation can be eliminated. See https://reactjs.org/docs/refs-and-the-dom.html

Reply:
That's true, I believe we can take advantage of reactive variables like state variables but good start on this.